app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -364,6 +374,20 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
     setIsModalVisible(!isModalVisible)
   }
 
+  const setSendingWalletDescriptor = (
+    walletDescriptor: WalletDescriptor<WalletCurrency>,
+  ) => {
+    if (
+      walletDescriptor.currency === WalletCurrency.Usd &&
+      paymentDetail.paymentType === PaymentType.Onchain
+    ) {
+      Alert.alert(LL.SendBitcoinScreen.walletDoesNotSupportOnchain())
```

Comment:
when would that be triggered? as we're setting btc wallet if this is an onchain address above by default

Reply:
It would be triggered by `click on wallet icon -> modal pops up allowing user to choose wallet -> user clicks on usd wallet -> alert is displayed`.

Reply:
I figured this would help the user understand better what is going on rather than just not having the option to switch wallets.